src/formatDate.js

```diff
@@ -48,9 +48,37 @@
  *
  * @returns {string}
  */
-
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
 
+  const calendar = {
+    formatYear(format) {
+      if (this.hasOwnProperty('YYYY') && format === 'YY') {
+        this['YY'] = this['YYYY'].slice(2, 4);
+        delete this['YYYY'];
+      } else if (this.hasOwnProperty('YY') && format === 'YYYY') {
+        this['YYYY'] = this['YY'] <= 20 ? '20' + this['YY'] : '19' + this['YY'];
+        delete this['YY'];
+      }
+    },
+  };
+
+  const toSeparator = toFormat[3];
```

Comment:
it should not be picked last item in array ? rather then 3 always ?

`const toSeparator = toFormat[toFormat.length -1];`